src/App.js

```diff
@@ -1,5 +1,99 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+import Car from "./Car.js";
+import { validateCarNameList, validateTryCount } from "./validation.js";
+import {
+  CAR_NAME_INPUT_MESSAGE,
+  TRY_COUNT_INPUT_MESSAGE,
+} from "./constants/inputMessage.js";
+import {
+  EMPTY_MESSAGE,
+  EXECUTE_RESULT_MESSAGE,
+  WINNER_MESSAGE,
+} from "./constants/outputMessage.js";
+
 class App {
-  async run() {}
+  #tryCount;
+  #cars;
+
+  constructor() {
+    this.#tryCount = 0;
+    this.#cars = [];
+  }
+
+  async run() {
+    let carNameList = await this.readCarName();
```

Comment:
```suggestion
    const carNameList = await this.readCarName();
```
`carNameList`가 재할당되는 것 같지는 않은데 `let` 키워드로 선언하신 이유가 있으신가요??

Reply:
헉 run에서 carNameList를 활용해 값을 새롭게 할당해주다가, 해당 기능을 삭제한 후 let을 const로 바꾸어주지 않았네요!
말씀해주신대로 const를 사용하는 것이 올바른 방법입니다.
꼼꼼한 발견 감사합니다! 😳